spec/system/support/cuprite_setup.rb

```diff
@@ -2,6 +2,8 @@
 
 require "capybara/cuprite"
 
+headless = ActiveModel::Type::Boolean.new.cast(ENV.fetch("HEADLESS", true))
```

Comment:
It's probably a good idea, but its not the purpose here. 
I think this should be removed from this PR, and open another one.

Reply:
You are right! I opened another [PR](https://github.com/openfoodfoundation/openfoodnetwork/pull/9545) :)